spec/concepts/governance.md

```diff
@@ -1,14 +1,107 @@
+description identifies the validator $v$ and the proposal, proves spend
+authority over a note recording $y$ `dPEN(v)`, and reveals the note's nullifier.
+Finally, it proves vote consistency $y = x_y + x_n + x_a + x_v$, produces a new
+note with $y$ `dPEN(v)`, and includes $\operatorname{Enc}_D(x_i)$, an encryption
+of the vote weights to the validators' decryption key.
+
+The proof statements in a `Vote` description establishing spend authority over
+the note are almost identical to those in a `Spend` description.  However, there
+are two key differences.  First, rather than proving that the note was included
+in a recent note commitment tree state, it always uses the root of the note
+commitment tree at the time that voting began, establishing that the note was
+not created after voting began.  Second, rather than checking the note's
+nullifier against the global nullifier set and marking it as spent, the
+nullifier is checked against a snapshot of the nullifier set at the time that
+voting began (establishing that it was unspent then), as well as against a
+per-proposal nullifier set (establishing that it has not already been used for
+voting).  In other words, instead of marking that the note has been spent in
+general, we only mark it as having been spent in the context of voting on a
+specific proposal.
+
+This change allows multiple proposals to be voted on concurrently, at the cost
+of linkability.  While the same note can be used to vote on multiple proposals,
+those votes, as well as the subsequent spend of the note, will have the same
+nullifier and thus be linkable to each other.  However, the `Vote` descriptions
+are shielded, so an observer only learns that two opaque votes were related to
+each other.
+
+We suggest that wallets roll over the note the first time it is used for voting
+by creating a transaction with `Vote`, `Spend`, and `Output` descriptions.  This
+mitigates linkability between `Vote` and `Spend` descriptions, and means that
+votes on any proposals created after the first vote are unlinkable from prior
+votes.
+
+# Counting Votes
+
+At the end of each epoch, validators collect the encrypted votes from each
+delegation pool, aggregate the encrypted votes into encrypted tallies and
+decrypt the tallies.  These intermediate tallies are revealed, because it is not
+possible to batch value flows over time intervals longer than one epoch.  In
+practice, this provides a similar dynamic as existing (transparent) on-chain
+governance schemes, where tallies are public while voting is ongoing.
+
+At the end of the voting period, the per-epoch tallies are summed.  For each
+validator $v$, the votes for each option are summed to determine the portion of
+the delegation pool that voted; the validator's vote acts as the default vote
+for the rest of the delegation pool.  Finally, these per-validator subtotals are
+multiplied by the voting power adjustment function $\theta_v(e)$ to obtain the
```

Comment:
clarifying question: is `$\theta_v(e)$` just a function that multiplies in the weights of each validator? or is more happening here?

Reply:
fixed in #124